client/components/fields/editor/EventAttachments.tsx

```diff
@@ -91,34 +91,23 @@ class EditorFieldEventAttachmentsComponent extends React.Component<IProps, IStat
         const value = get(this.props.item, field, this.props.defaultValue ?? []);
 
         return (
-            <ToggleBox
-                ref={this.props.refNode}
-                title={this.props.label ?? gettext('Attached Files')}
-                isOpen={false}
-                onOpen={this.onOpen}
-                scrollInView={true}
-                hideUsingCSS={true} // hideUsingCSS so the file data is kept on hide/show
```

Comment:
Come to think of it, this needs to be addressed as well.
If a user uploads a file then collapses the ToggleBox, then I believe the file data is lost.
Could you please test this with your solution please?